docs/gallery/articles/fraud_detection/index.md

```diff
@@ -4,38 +4,64 @@ category: finance
 data-keywords: dashboard vizelement layout chart ai multi-page classification enterprise
 short-description: A Taipy Application that analyzes credit card transactions to detect fraud.
 order: 20
-img: fraud_detection/images/fraud_threshold.png
+img: fraud_detection/images/transactions_page.png
 hide:
     - toc
 ---
-A Taipy Application that analyzes credit card transactions to detect fraud.
+
+A Taipy Application to analyze credit card transactions, detect fraud, and collaborate with other 
+application users.
 
 !!! note "Taipy Enterprise edition"
 
     Taipy provides robust, business-focused applications tailored for enterprise environments. To
     maintain standards of security and customization, these applications are proprietary like this
-    application. If you’re looking for solutions that are immediately deployable and customizable to
+    application. If you're looking for solutions that are immediately deployable and customizable to
```

Comment:
Hahaha! How much time did you spend on it?

Reply:
In fact, this was already in the documentation. I just fixed it